community/pike/APKBUILD

```diff
@@ -0,0 +1,188 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+_pkgname=Pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="GPL-2.0-or-later"
+options="!check" # there are no tests for those packages
+depends="gmp nettle nettle-utils readline"
+makedepends="gmp-dev
+	bzip2-dev
+	freeglut-dev
+	gdbm-dev
+	giflib-dev
+	giflib-utils
+	gtk+2.0
+	libglade-dev
+	libsm-dev
+	mesa-gl
+	nettle
+	nettle-dev
+	nettle-utils
+	pcre-dev
+	sdl-dev
+	sdl_mixer-dev
+	sqlite-dev
+	sqlite-libs
+	readline
+	zlib-dev
+	librsvg-dev"
+install=
+subpackages="$pkgname-doc $pkgname-full"
+source="http://pike.lysator.liu.se/pub/$pkgname/all/$pkgver/$_pkgname-v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-v$pkgver"
+
+_full_deps="$pkgname=$pkgver-r$pkgrel $pkgname-doc=$pkgver-r$pkgrel"
+for _name in bzip2 gdbm gl gtk image pcre sdl svg sqlite
+do
+	subpackages="$subpackages $pkgname-$_name"
+	_full_deps="$_full_deps $pkgname-$_name=$pkgver-r$pkgrel"
+done
+
+build() {
+	cd "$builddir"
+	make CONFIGUREARGS=" \
+		--prefix=/usr \
+		--disable-make_conf \
+		--disable-noopty-retry \
+		--without-cdebug \
+		--without-bundles \
+		--without-ssleay \
+		--with-gmp \
+		--with-crypt \
+		--with-bignums \
+		--without-rtldebug \
+		--with-Bz2 \
+		--without-fftw \
+		--without-_Ffmpeg \
+		--with-gdbm \
+		--without-GTK1 \
+		--with-GTK2 \
+		--with-jpeglib \
+		--without-Kerberos \
+		--without-msql \
+		--without-mysql \
+		--without-Odbc \
+		--without-oracle \
+		--with-GL \
+		--with-GLUT \
+		--with-_Regexp_PCRE \
+		--without-libpdf \
+		--without-sass \
+		--with-sane \
+		--with-SDL \
+		--with-SDL_Mixer \
+		--with-svg \
+		--with-tifflib \
+		--without-freetype \
+		--without-vcdiff \
+		--with-_Image_WebP \
+		--with-zlib"
+	PATH="$builddir/bin:$PATH" make doc
+}
+
+package() {
+	cd "$builddir"
+	make INSTALLARGS="--traditional" buildroot="$pkgdir" install
+}
+
+_install() {
+	local _arg
+	for _arg in "$@"; do
+		[ -e "$_arg" ] || continue
+		local bn=$(basename "$_arg")
+		local dn=$(dirname "$_arg")
+		mkdir -p "$subpkgdir/$dn"
+		mv "$_arg" "$subpkgdir/$dn/$bn"
+	done
+}
+
+bzip2() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname support"
+	depends="pike"
+	_install usr/lib/pike/modules/*Bz2.*
+}
+
+gdbm() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GDBM support"
+	depends="pike gdbm"
```

Comment:
`gdbm` is not needed in depends here. See https://pkgs.alpinelinux.org/contents?file=libgdbm.so.4&path=&name=&branch=edge&arch=x86_64